app/gateway/resource-sizing-guidelines.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "{{site.base_gateway}} resource sizing guidelines"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+min_version:
+    gateway: 3.4.x
+
+tags:
+    - performance
+
+breadcrumbs:
+    - /gateway
+
+description: "See Kong's recommended resource allocation sizing guidelines for {{site.base_gateway}} based on configuration and traffic patterns."
+
+related_resources:
+  - text: Performance benchmark
+    url: /
+  - text: Performance benchmark how to
+    url: /
```

Comment:
```suggestion
  - text: Guidelines for establishing a performance benchmark
    url: /gateway/performance/establish-performance-benchmark/
```
Since this will be a reference and not a how-to, rephrasing to sound that way.